dev/ci/gitlab.bat

```diff
@@ -43,8 +70,46 @@ IF DEFINED WIN_CERTIFICATE_PATH (
   )
 )
 
+ECHO "Finished Artifact Creation"
+TIME /T
+
+CALL :CleanupFolders
+
+ECHO "Finished Cleanup"
+TIME /T
+
 GOTO :EOF
 
+:ZipLogfiles
+  ECHO Zipping logfiles for artifact upload
```

Comment:
Originally I did it like this to not clutter up the git folder more than necessary (artifacts can only be uploaded from the git folder), but I meanwhile found it a bit inconvenient. I will create an artifacts folder and put everything in there.

Reply:
@SkySkimmer : I changed this and the files are more accessible now. One disadvantage is that the file change dates are gone this way. This can be useful information. Any ideas?

I could zip the flagfiles, which mostly exist to serve as timestamps, and keep the rest as copied files or I could dump a time stamp in the flagfiles. Of cause one has the information now also in the main log.

Another thing is that I probably I should rename the extensions to .log.txt and .err.txt. Otherwise they cannot be viewed in the browser.